doc/devdocs/modules/launcher/plugins/registry.md

```diff
@@ -0,0 +1,96 @@
+# Registry Plugin
+
+The registry plugin allow the users to search into the Windows registry.
```

Comment:
"The registry plugin allow**s**  users to **search the** Windows registry." sounds better